app/assets/javascripts/components/generic_object/main-custom-button-form-component.js

```diff
@@ -126,12 +126,16 @@ function mainCustomButtonFormController(API, miqService, $q, $http) {
 
   vm.saveClicked = function() {
     var saveMsg = sprintf(__('%s "%s" has been successfully saved.'), vm.entity, vm.customButtonModel.name);
-    vm.saveWithAPI('put', '/api/custom_buttons/' + vm.customButtonRecordId, vm.prepSaveObject(), saveMsg);
```

Comment:
forgot to remove `saveWithAPI`? ;)

This change also drops the logic when `vm.customButtonGroupRecordId` is true, was that intentional?